docs/admin/cluster-management.md

```diff
@@ -169,8 +169,8 @@ This is an infrequent event, but it requires careful management. There is a sequ
    1. Update existing objects in the storage to new version by running `cluster/update-storage-objects.sh`.
    1. Turn off the old API version.
 
+<a name="Turn-on-or-off-an-api-version-for-your-cluster"></a>
```

Comment:
Yeah - I added the anchor "manually" because I thought that was the expected way to build out that feature. I will edit / recommit / etc. if someone tells me how I should do this - NP.